sova/supervisor/dependabot.py

```
@@ -0,0 +1,562 @@
+            result = await _process_pr(
+                dpr,
+                repo=repo,
+                github_user=github_user,
+                config=config,
+                project_dir=project_dir,
+                notification_config=notification_config,
+            )
+            results.append(result)
+        except Exception:
+            log.warning("dependabot.process_error", pr=dpr.number, exc_info=True)
+            results.append(MergeResult(dpr.number, dpr.title, "error", "unhandled exception"))
```

Ruff (0.16.1)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)